clouddriver-ecs/src/integration/README.md

```diff
@@ -8,3 +8,22 @@ To manually run the gradle task for these tests:
 ```bash
 $> ./gradlew :clouddriver-ecs:integrationTest
 ```
+
+## Guidance for modifying this package
+
+### Adding new tests
+
+New Amazon ECS provider features of significant scope should include an integration test which exercises new functionality.
+Examples of qualifying changes include (but are not limited to):
+
+* Implementing a new atomic operation
+* New forking logic in how `CreateServerGroup` functions or is validated (especially re: broadly impactful settings like `networkMode`, launch type, load balancing, or application autoscaling)
+* Adding a new controller for a new type of resource
+
+### Changing existing tests
+
+In general, existing test cases should function as-is after new contributions to ensure existing features continue to function as expected.
+Possible exceptions to this guidance may include:
+
+* Updates to internal implementation details (required `@Beans`, etc.) that don't effect operation success or API response content
+* Adding and asserting on *new* data in a `clouddriver` API response
```

Comment:
Can we specify an example for how JSON modification might be needed to accommodate new changes after this line?
e.g. If we add support for target group mappings then legacy target group parameter should be updated to start accepting new target group mappings which has few extra parameters.

Reply:
> e.g. If we add support for target group mappings then legacy target group parameter should be updated to start accepting new target group mappings which has few extra parameters.

I'm a little confused by what you mean. Adding new fields/info to a test's request payload (which we are storing as JSON) is not something we should be doing on _existing_ tests (which this section is referring to), b/c that means we are requiring new information to fulfill old test cases and that would break people with existing deployments/pipelines.

Reply:
Correct, makes sense. Then we can just specify this example as a part of the 1st section for new tests.